elixir/apps/api/test/api/client/channel_test.exs

```diff
@@ -262,15 +262,18 @@ defmodule API.Client.ChannelTest do
       global_relay =
         Fixtures.Relays.create_relay(
           group: global_relay_group,
-          ipv6: nil,
```

Comment:
We need a bit more edge cases tested:
1. Self-Hosted relays
2. Different routing settings hide and show relays from the list